calico_containers/tests/st/test_add_ip.py

```diff
@@ -0,0 +1,69 @@
+from sh import ErrorReturnCode
+from functools import partial
+
+from test_base import TestBase
+from docker_host import DockerHost
+
+
+class TestAddIp(TestBase):
+    def test_add_ip(self):
+        host = DockerHost('host')
+
+        calicoctl = "/code/dist/calicoctl %s"
+        host.execute(calicoctl % "node --ip=172.17.8.10")
```

Comment:
The author of the original shell script. Should we default to `127.0.0.1` in single-host tests?